src/cluster.c

```diff
@@ -6020,7 +6020,8 @@ int clusterRedirectBlockedClientIfNeeded(client *c) {
     if (c->flags & CLIENT_BLOCKED &&
         (c->btype == BLOCKED_LIST ||
          c->btype == BLOCKED_ZSET ||
-         c->btype == BLOCKED_STREAM))
+         c->btype == BLOCKED_STREAM||
```

Comment:
stream is possibly also a special case, since unlike list and zset, a client can be blocked on a stream key that **exists**

Reply:
not sure stream requires any special handling... it doesn't matter if the exists or not, the only thing that matters is the hash slot